Fix quick facts overflow

bhumika-gupta opened the pull request:
<img width="1253" alt="Screen Shot 2021-08-19 at 9 24 02 PM" src="https://user-images.githubusercontent.com/63598253/130164321-1e2ddd99-394b-477e-a76d-eca8a21afca6.png">

Chrome browser 1440 x 764

bhumika-gupta committed:
Remove top property for BackArrow media query

bhumika-gupta committed:
Change width of TitleSection media query

bhumika-gupta committed:
Change left positioning of BackArrow media query

bhumika-gupta committed:
Change top positioning of BackArrow media query

bhumika-gupta committed:
Fix top positioning for BackArrow media query

bhumika-gupta committed:
Merge branch 'main' of https://github.com/PuMProfessionals/pump-web-v2 into adjust-title-component

bhumika-gupta committed:
Merge branch 'main' of https://github.com/PuMProfessionals/pump-web-v2 into adjust-title-component

bhumika-gupta committed:
Change positioning of Back Arrow

bhumika-gupta committed:
Merge branch 'main' of https://github.com/PuMProfessionals/pump-web-v2 into adjust-title-component

bhumika-gupta committed:
Remove  commented out code

bhumika-gupta committed:
Change positioning of back arrow

bhumika-gupta committed:
Change positioning of back arrow

bhumika-gupta committed:
Change positioning of back arrow on normal screen

bhumika-gupta committed:
Merge branch 'main' of https://github.com/PuMProfessionals/pump-web-v2 into adjust-title-component

bhumika-gupta committed:
Add new div for arrow and title

bhumika-gupta committed:
Fix small edit

bhumika-gupta committed:
Fix merge conflicts

bhumika-gupta committed:
Change width of TopSection

bhumika-gupta committed:
Fix overflow on cards

bhumika-gupta committed:
Fix merge conflicts

xsharonhe reviewed (commented):
Merge conflicts are still existent you can tell by the <<< arrows in Title

hirundinidae committed:
Fix merge conflicts

Hope I chose the right one to keep in title width. Not changing width
based on isLeftAligned looks weird (too far to the left)

hirundinidae reviewed (approved):
@bhumika-gupta I removed the remaining `>>>>>` lines and the quick fact cards look fine to me
